src/go/cluster-to-redpanda-migration/tests/data/00-console.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Console
+metadata:
+  name: console-dedicated
+spec:
+  server:
+    listenPort: 8080
+  schema:
+    enabled: true
```

Comment:
There should be some schema registry configuration in the `00-console-rp_output.yaml`